projects/example-app/src/app/core/effects/router.effects.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { createEffect } from '@ngrx/effects';
+import { Router, NavigationEnd, ActivatedRoute } from '@angular/router';
```

Comment:
Move all `@angular` imports to the top of the file, following by `rxjs` and `@ngrx`